src/shared/components/plots/BoxScatterPlot.tsx

```diff
@@ -452,7 +452,7 @@ export default class BoxScatterPlot<D extends IBaseBoxScatterPlotPoint> extends
                                                   verticalAnchor={this.props.horizontal ? undefined : "start"}
                                                   textAnchor={this.props.horizontal ? undefined : "start"}
                                   />}
-                axisLabelComponent={<VictoryLabel dy={this.props.horizontal ? 35 : this.biggestCategoryLabelSize + 24}/>}
+                axisLabelComponent={<VictoryLabel dy={this.props.horizontal ? 35 : this.biggestCategoryLabelSize + 14}/>}
```

Comment:
are you sure this is generally better? if so, okay, but if not maybe you can handle it with a prop